src/main/scala/viper/gobra/reporting/DefaultErrorBackTranslator.scala

```diff
@@ -64,22 +64,23 @@ object DefaultErrorBackTranslator {
       case vprrea.NegativePermission(CertainSource(info)) =>
         NegativePermissionReason(info)
       //      case vprrea.InvalidPermMultiplication(offendingNode) =>
-      //      case vprrea.MagicWandChunkNotFound(offendingNode) =>
-      //      case vprrea.NamedMagicWandChunkNotFound(offendingNode) =>
-      //      case vprrea.MagicWandChunkOutdated(offendingNode) =>
+      case vprrea.MagicWandChunkNotFound(CertainSource(info)) =>
```

Comment:
why are `NamedMagicWandChunkNotFound` and `MagicWandChunkOutdated` not needed?

Reply:
These error reasons are not longer available in Silver